modules/grad_killhouse/functions/builds/fn_basic.sqf

```diff
@@ -0,0 +1,729 @@
+params ["_kind"];
+
+_pos = [23800.5,16012.6,0];
+_dir = 0;
+_array = [];
```

Comment:
Hier kannst du auch 

```sqf
_array = [
    ["Land_Shoot_House_Wall_F",(_pos vectorAdd [1.00025,2.00375,0]),(_dir + 180)] call grad_killhouse_fnc_createObj,
    ["Land_Shoot_House_Wall_F",(_pos vectorAdd [3.00025,2.00375,0]),(_dir + 180)] call grad_killhouse_fnc_createObj,
    ["TargetP_Inf_F",(_pos vectorAdd [2,3.83957,0]),(_dir + 180), _kind] call grad_killhouse_fnc_createTarget,
    ["Land_Shoot_House_Wall_F",(_pos vectorAdd [5.00025,2.00375,0]),(_dir + 180)] call grad_killhouse_fnc_createObj,
    ....
];
```

machen, dann sparst du dir die 563 `pushBacks`.

Reply:
stimmt, danke :D

Reply:
Tried, does not work, going back to the old pushBack